<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormMeasures</class>
 <widget class="QMainWindow" name="FormMeasures">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Набор измерений</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Набор</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="verticalSpacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>6</number>
         </property>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLineEdit" name="lnTypesFilter">
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>Наберите название набора</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="listDataset">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="tabKeyNavigation">
              <bool>false</bool>
             </property>
             <property name="defaultDropAction">
              <enum>Qt::MoveAction</enum>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="movement">
              <enum>QListView::Static</enum>
             </property>
             <property name="flow">
              <enum>QListView::TopToBottom</enum>
             </property>
             <property name="isWrapping" stdset="0">
              <bool>false</bool>
             </property>
             <property name="layoutMode">
              <enum>QListView::SinglePass</enum>
             </property>
             <property name="viewMode">
              <enum>QListView::ListMode</enum>
             </property>
             <property name="uniformItemSizes">
              <bool>false</bool>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
             <property name="selectionRectVisible">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnAddSet">
             <property name="text">
              <string>Добавить набор</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Измерения</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="verticalSpacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>6</number>
         </property>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLineEdit" name="lnMeasuresFIlter">
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>Наберите название измерения</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="listMeasures">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnAddMeasure">
             <property name="text">
              <string>Добавить измерение</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Информация</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="verticalSpacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>6</number>
         </property>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QTableView" name="tblInfo">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="tabKeyNavigation">
              <bool>false</bool>
             </property>
             <property name="showDropIndicator" stdset="0">
              <bool>false</bool>
             </property>
             <attribute name="horizontalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnEditMeasure">
             <property name="text">
              <string>Редактировать измерение</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnAccept">
        <property name="text">
         <string>Выбрать набор</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>823</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Измерения</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>Добавить</string>
     </property>
     <addaction name="actionAddSet"/>
     <addaction name="actionAddMeasure"/>
    </widget>
    <addaction name="menu_4"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Правка</string>
    </property>
   </widget>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Добавить набор</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Добавить измерение</string>
   </property>
  </action>
  <action name="actionAddSet">
   <property name="text">
    <string>Набор</string>
   </property>
  </action>
  <action name="actionAddMeasure">
   <property name="text">
    <string>Измерение</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
